blazor/datagrid/excel-exporting.md

```diff
@@ -366,6 +366,149 @@ To apply theme in exported Excel, define the **theme** in export properties.
 
 > By default, material theme is applied to exported excel document.
 
+### Customize background color for Grid in Excel Exported document
+
+You can set the background color for Grid in the exported document using the [Theme](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html#Syncfusion_Blazor_Grids_ExcelExportProperties_Theme) property of [ExcelExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html) class.
```

Comment:
You can set the background color for a Grid in the exported document using the [Theme]